build.gradle

```diff
@@ -18,13 +18,11 @@ buildscript {
     repositories {
         maven { url "https://plugins.gradle.org/m2/" }
     }
-    dependencies {
-        classpath 'gradle.plugin.org.jetbrains:gradle-intellij-plugin:0.0.43'
-    }
 }
 
 plugins {
     id 'net.researchgate.release' version '2.3.4'
+    id 'org.jetbrains.intellij' version '0.1.5'
```

Comment:
This has been a valid way to import when we first set this up. But it did
cause breakages, I think related to evaluation order of stuff or something.
(sorry my memory is not so good) @loosebazooka might remember better.

On Thu, Jun 2, 2016 at 3:42 PM, Etan Shaul <notifications@github.com> wrote:

> In build.gradle
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/708#discussion_r65605102>
> :
>
> >  }
> >
> >  plugins {
> >      id 'net.researchgate.release' version '2.3.4'
> > +    id 'org.jetbrains.intellij' version '0.1.5'
>
> Its the new way to do it since gradle 2.1. I only recently upgraded the
> gradle wrapper on the project.
>
> Its the version of the plugin that's breaking stuff. Not sure why yet.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/708/files/d18beb8b378475ba4fb10ea67ad9eb5f6c0a4d32#r65605102>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHf5HRtQvun-sUdVLsGSiWpSL1pbqkkLks5qHzIJgaJpZM4Is2Xx>
> .
>